controllers/resources/bentorequest_controller.go

```diff
@@ -1758,30 +1759,50 @@ fi
 
 mkdir -p {{.ModelDirPath}}
 url="{{.ModelDownloadURL}}"
-echo "Downloading model {{.ModelRepositoryName}}:{{.ModelVersion}} to /tmp/downloaded.tar..."
-if [[ ${url} == s3://* ]]; then
-	echo "Downloading from s3..."
-	aws s3 cp ${url} /tmp/downloaded.tar
-elif [[ ${url} == gs://* ]]; then
-    echo "Downloading from GCS..."
-    gsutil cp ${url} /tmp/downloaded.tar
+
+cleanup() {
+	echo "Cleaning up..."
+	rm -rf /tmp/model
+	rm -f /tmp/downloaded.tar
+}
```

Comment:
‌This line needs to follow immediately after this function, otherwise the defer effect won't be achieved:

```bash
trap cleanup EXIT
```